internal_metrics/dimensions.yaml

```diff
@@ -0,0 +1,74 @@
+line:
+  brief: Line number of the code block instrumented.
+  description: Line number of the code block instrumented..
+  example: 170
+
+filename:
+  brief: Filename of the source code file.
+  description: File name of the source code file.
+  example: agg_core.cc
+
+k8s.cluster.name:
+  brief: Kubernets Cluster name.
+  description: Kubernetes Cluster name.
+  example: staging.
+
+az:
+  brief: availability zone
+  description: availability zone
+  example: us-east-1a
+
+error:
+  brief: error type
+  description: error type
+  example: fetched/unknown
+
+kernel:
+  brief: Linux kernel version
+  description: Linux kernel version
+  example: 5.4.219-126.411.amzn2.x86_64
+
+os:
+  brief: Operating Systems
+  description: Name of Operating System
+  example: Linux
+
+image_version:
+  brief: Kubernetes pod version label of the workload.
+  description: If the associated metric comes from a monitored Kubernetes cluster running the k8-collector, the metric may note the source and destination versions that the traffic being measured moved between.  The version label is optionally set by the administrator configuring Kubernetes.  Sometimes referred to as the 'service version.'  May be found in the docker (pod) image or Kubernetes metadata.
+  example: latest, -, v0.3.6
+
+module:
+  brief: name of the Reducer core
+  description: logging
+
+peer:
+  brief: peer module
+  description: See module
+
+name:
+  brief: Name of instrumented codeblock.
+  description: Name of instrumented codeblock.
+  example: OtlpGrpcFormatterFormatLabelsChanged 
+
+c_host:
+  brief: Collector host machine name.
+  description: Collector host machine name.
+  example: ip-192-168-110-244.ec2.internal 
+
+c_type:
+  brief: Collector type
+  description: Collector type
+  example: 1
```

Comment:
Do we want to define here what the type numbers equate to?  https://github.com/open-telemetry/opentelemetry-ebpf/blob/main/common/client_type.h

Reply:
+1